webv2/web.go

```diff
@@ -777,98 +777,95 @@ func setParentTable(w http.ResponseWriter, r *http.Request) {
 }
 
 func parentTableHelper(table string, update bool) *TableInterleaveStatus {
-	tableInterleaveStatus := &TableInterleaveStatus{Possible: true}
```

Comment:
getting same issue twice after changing pk order : https://drive.google.com/open?id=1j0Og0EnjsO2-nfaVUGZfM5SOye3e4T3A&authuser=0&resourcekey=0-WbU2fin4SYQWyLxMSfp-OA&usp=drive_link

Reply:
Updated. Please review it.